### PR TITLE
Restructure .NET sdk landing page

### PR DESCRIPTION
**Pull request by hhunter-ms**
Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

# Description

Per discussion w @msfussell - restructuring landing page to be more navigable and consistent with JS and Python structure.

## Issue reference

PR will close: #880 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: daprdocs/content/en/dotnet-sdk-docs/_index.md [hunk 1]
```diff
@@ -9,6 +9,24 @@ no_list: true
 
 Dapr offers a variety of packages to help with the development of .NET applications. Using them you can create .NET clients, servers, and virtual actors with Dapr.
 
+## Installation
+
+To get started with the .NET SDK, install the Dapr .NET SDK package:
+
+```sh
+dotnet install dapr
```
Comment (halspang) [inline daprdocs/content/en/dotnet-sdk-docs/_index.md:17]: Where did this command come from/are we just trying to use the packages or are we hoping for more here?

**Review by halspang — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/dotnet-sdk-docs/_index.md (hunk 1 above)
Comment (hhunter-ms) [reply daprdocs/content/en/dotnet-sdk-docs/_index.md:17]: I used the wrong command - my bad. Ok if I use this one instead?:

> `dotnet add package Dapr.Client --version 1.9.0`

Comment (halspang) [reply daprdocs/content/en/dotnet-sdk-docs/_index.md]: @hhunter-ms  - Adding the version means we'll have to update the doc with each release. Can we keep it versionless and say that that command will add the latest instead?

Comment (hhunter-ms) [reply daprdocs/content/en/dotnet-sdk-docs/_index.md]: @halspang that works for me, I'll update

**Review by halspang — APPROVED** [review]

**Pull request merged**